<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="test">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionCode_path"/>
   </widget>
   <widget class="QMenu" name="menuCode">
    <property name="title">
     <string>Code</string>
    </property>
    <addaction name="actionCocos2dx"/>
    <addaction name="actionUnity"/>
    <addaction name="actionPhp"/>
   </widget>
   <addaction name="menuSetting"/>
   <addaction name="menuCode"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCode_path">
   <property name="text">
    <string>code path...</string>
   </property>
  </action>
  <action name="actionCocos2dx">
   <property name="text">
    <string>cocos2dx</string>
   </property>
  </action>
  <action name="actionUnity">
   <property name="text">
    <string>unity</string>
   </property>
  </action>
  <action name="actionPhp">
   <property name="text">
    <string>php</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
